docs/integrations/bitbucket.rst

```diff
@@ -0,0 +1,28 @@
+Bitbucket
+=========
+
+Sentry integrates with both Bitbucket Issues and Pipelines.
+
+
+Bitbucket Issue Creation from Sentry
+````````````````````````````````````
+
+Create issues in Bitbucket based on Sentry issues.
+
+1. Go to the project settings page in Sentry that you'd like to link with Bitbucket
+2. Click All Integrations, find the Bitbucket integration in the list, and click configure
+3. Click Enable Plugin
+4. Fill in the required information and save
+5. The option to create Bitbucket issues will be displayed from your Sentry issue pages
+
+Release Creation with Bitbucket Pipelines
```

Comment:
*Set up release creation with Bitbucket Pipelines*

Is "creation" really the right word? You're not creating releases, right?